documentation/modules/overview/con-configuration-points-bridge.adoc

```diff
@@ -11,6 +11,18 @@ Kafka Bridge consumer and producer configuration is standard, as described in th
 
 HTTP-related configuration options set the port connection which the server listens on.
 
+[discrete]
+== CORS
+
+The Kafka Bridge supports the use of Cross-Origin Resource Sharing (CORS).
+CORS is a HTTP mechanism that allows browser access to selected resources from more than one origin.
```

Comment:
Does the intended audience understand what is meant by an origin, in the context of HTTP requests? Consider adding an example, such as "for example, a domain".

Reply:
Added:

> , for example, resources on different domains.